crossdock/client/tchclient/thrift.go

```diff
@@ -21,11 +21,16 @@
 package tchclient
 
 import (
+	"reflect"
+	"strings"
 	"time"
 
 	"github.com/yarpc/yarpc-go/crossdock-go"
+	"github.com/yarpc/yarpc-go/crossdock/client/gauntlet"
+	"github.com/yarpc/yarpc-go/crossdock/client/pointer"
 	"github.com/yarpc/yarpc-go/crossdock/client/random"
 	"github.com/yarpc/yarpc-go/crossdock/thrift/gen-go/echo"
+	"github.com/yarpc/yarpc-go/crossdock/thrift/gen-go/gauntlet_apache"
```

Comment:
the leaf package name cannot contain hypens, but directories can. The default output from Apache thrift creates `gen-go` hence the hyphen (internally, we prefer to use `.gen/go`).

Ideally package names do not have underscores, but this comes from the filename.

Idiomatic go code doesn't use hypens or underscore normally.